pkg/controller/resolvedofferings/computeclass.go

```diff
@@ -134,11 +143,13 @@ func resolveComputeClassForContainer(req router.Request, appInstance *v1.AppInst
 		}
 		def := parsedMemory.Def.Value()
 		memory = &def
+		cpuQuantity := computeclasses.CalculateCPU(*cc, *resource.NewQuantity(*memory, resource.BinarySI))
+		cpu = z.Pointer(cpuQuantity.MilliValue())
```

Comment:
This does not seem correct to me, as it only calculates the cpu if the user did not override the memory in the acorn image or with CLI flags and is just leaving it to the compute class' default memory. I think we want to calculate the cpu in every circumstance, right? And I don't think there is a way for the user to override the CPUScaler since it is hidden from them. So we should just have a separate if statement after this whole `if else if` series that does `if cc != nil` and then calculates the CPU based on the scaler from the CC.

Reply:
Yep, you're right. I've gone ahead and fixed this, funny enough this was a last minute change and used to be what you suggested so I should've gone with my original gut feeling here lol.